### DISCUSSION TITLE
Creating queries that run every n hours

### DISCUSSION THREAD

**Discussion by rishavs** [planetscale/discussion#559] [Q&A] [answered]
Hi

I want to hydrate some aggregated tables using the data from base tables, and the aggregation needs to happen every hour.
Since PS doesn't supports triggers, how should I set this up?

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @rishavs,

This would depend on what framework your application is built in. You'd need some sort of scheduled job or cron to trigger the code hourly.

For example, one of our apps uses Rails. We use Sidekiq to schedule hourly jobs to run.

**Comment by rishavs** [comment]
Thanks.
